### DISCUSSION TITLE
List view that is agnostic to a time period

### DISCUSSION THREAD

**Discussion by Fernando-Magnabosco** [vkurko/calendar#326] [Q&A]
Hi there! I'd like to know if there's a way to implement a list view that is agnostic of the period of time. I've seen that the available views are [these](https://github.com/vkurko/calendar?tab=readme-ov-file#view), but all the list views (actually all the views) are bonded to a period of time (e.g.  'list{day}', 'list{week}', and so on). What I need is a view that renders all the events on the `events` array, ordered by date, but not grouped by date.

**Comment by letyrsl** [comment]
I agree this could be an amazing and very useful feature... 
Specially for an event search implementation without having date range limitations

**Reply by Fernando-Magnabosco** [reply]
thank u!

**Comment by vkurko** [comment]
I'm not sure about this idea. After all, Event Calendar is primarily a calendar displaying events for a selected period of time.
